packages/components/src/components/form/SegmentedButton/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './SegmentedButton';
```

Comment:
we should export the types too, at least the one _relevant_ to the component usage